### DISCUSSION TITLE
Error in brandnew installation

### DISCUSSION THREAD

**Discussion by quimqu** [lllyasviel/Fooocus#1467] [General]
Hello,

I just installed a brand new installation of Manjaro (everything updated) and followed the setps to install Fooocus. Everthing looks fine, the front opens but if I try to generate an image, I just get an Error without any clues of what type of error it is. I tried in Chrome and Firefox. I have a RTX 3060.... Any idea what should I look for?

I just activated the swap of 40GB also... the main problem is that I cannot see the error message so I have no idea what is going on.

Thank you very much!

**Comment by quimqu** [comment]
OK, no nvidia drivers... sorry

**Reply by calvintirrell** [reply]
@quimqu : Was your issue that you had no Nvidia drivers installed or that you had to uninstall Nvidia drivers?

**Reply by quimqu** [reply]
Hi, I had no drivers installed. The error raised in the console, just before opening the browser with the trontend.

**Reply by calvintirrell** [reply]
Gotcha, I just realized I was following old/bad directions and need to re-install from scratch to solve my problem. Thanks.
